calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -296,7 +296,7 @@ kubectl patch networks.operator.openshift.io cluster --type merge -p '{"spec":{"
 If you cannot disable `kube-proxy` (for example, because it is managed by your Kubernetes distribution), then you _must_ change Felix configuration parameter `BPFKubeProxyIptablesCleanupEnabled` to `false`. This can be done with `kubectl` as follows:
 
 ```
-kubectl patch felixconfiguration.p default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'
+calicoctl patch felixconfiguration default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'
```

Comment:
```suggestion
kubectl patch felixconfiguration default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'



```

Reply:
The last time I executed this command, it returned the following error:
```
# kubectl get felixconfiguration.p default
error: the server doesn't have a resource type "felixconfiguration"
```

Reply:
@lwr20 Can you weigh in here? Is this indeed one that kubectl can't handle? Are the related fixes in #677 wrong?

Reply:
On my kind cluster with calico installed:

Reply:
Maybe the issue was `.p`. I will accept your commit suggestion, @ctauchen.